file-formats/evto/examples/z_aff_example.evto.json

```
@@ -0,0 +1,11 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example Event Object.",
+    "originalLanguage": "en"
+  },
+  "majorVersion": 1,
+  "minorVersion": 0,
+  "patchVersion": 4,
```

Just an idea: Another option would be to have something like this, wouldn't it? For me a semantic version is better readable, but you would need to do more validations and parsing it into several ABAP fields...

```suggestion
  "semanticVersion": "1.0.4",
```

What do you think?

---

Yes, I thought about this but decided against it for consistency. EVTO is quite near to the Event Binding (EVTB) and Service Binding (SRVB) repository objects. Both have the semantic version input fields as separate fields.